### ISSUE TITLE
Great General and Great Admiral Points Yields don't have a description text!

### ISSUE THREAD

**Issue by TechpriestEnginseer** [LoneGazebo/Community-Patch-DLL#884]
![image](https://cloud.githubusercontent.com/assets/13937047/11614361/651e368c-9bf5-11e5-8a67-ebf6cbf6d21c.png)

Although it says so in the description, when you apply these yields to any buildings, it returns TXT_KEY_YIELD_GREAT_GENERAL/ADMIRAL_POINTS.

**Comment by LoneGazebo** [comment]
That's because you need to add the TXT file yourself - as the CP/CBP do not use that mechanic, I don't see the need for a TXT file.

G

**Comment by TechpriestEnginseer** [comment]
But won't two TXT_KEY_YIELD_GREAT_GENERAL/ADMIRAL_POINTS conflict with each other from two different mods?

**Comment by LoneGazebo** [comment]
Eh?

**Comment by TechpriestEnginseer** [comment]
![image](https://cloud.githubusercontent.com/assets/13937047/11619006/4a4f83b2-9c57-11e5-8007-84dcd75bea03.png)

Hypothetical scenario: I am a player.

I have installed a mod that adds a building that grants great general points. This mod's author has added in a TXT file.

![image](https://cloud.githubusercontent.com/assets/13937047/11619019/81e3dd8c-9c57-11e5-920a-a7d3682a6987.png)
I've also installed another mod that gives me a building that grants great general points. The mod's author has also added in another TXT file. Knowing me I love great generals and I would think they would merge together since they would have no building conflict right?

![image](https://cloud.githubusercontent.com/assets/13937047/11619064/02392b4a-9c58-11e5-90e1-6094933ca724.png)
